ui/partials/logviewer/lvLogLines.html

```diff
@@ -3,8 +3,9 @@
 
 <div ng-repeat="lv_line in displayedLogLines"
      ng-class="{'text-danger': (lv_line.hasError == true),
+                'text-selected': (lv_line.selected == true),
                 'lv-selected-lines': (displayedStep.order === lv_line.index)}"
```

Comment:
So it turns out `lv-selected-lines` is not used any more. Let's use that line instead of `text-selected`, and make it singular:

`'lv-selected-line': (lv_line.selected == true)`

And instead of applying it there, let's apply it with ng-class to the existing span on L10. 

The user also needs to be able to click or drag-select portions of a line without triggering your new highlight function. We should also not include line numbers during multi-line selection of the error text. You'll want to ensure that is the case here by adding an adjacent div or span on L10 which we can also style separately (I haven't tested the multi-line selection for this yet):

eg. `<div class="lv-line-num">{{$index}}</div>`

So we only highlight the line if they click on the left hand line-number in a visually isolated column. Like Github:

![githubreference](https://cloud.githubusercontent.com/assets/3660661/9746492/4a1b4838-5646-11e5-8274-0c319a5d223b.jpg)

Having that separate div or span on L10 below will also allow us to style it independently and apply a new `:hover`,  `background` and `border-right` attributes, like Github above.